.github/workflows/release-check.yml

```diff
@@ -0,0 +1,44 @@
+name: Release Checker
+
+on:
+  pull_request_target:
+    paths: [
+      "dispatch_examples/greeter/Cargo.toml",
+      "frc42_dispatch/Cargo.toml",
+      "frc42_dispatch/hasher/Cargo.toml",
+      "dispatch_examples/greeter/Cargo.toml",
+      "frc42_dispatch/Cargo.toml",
+      "frc42_dispatch/hasher/Cargo.toml",
+      "frc42_dispatch/macros/Cargo.toml",
+      "frc46_token/Cargo.toml",
+      "frc53_nft/Cargo.toml",
+      "fvm_actor_utils/Cargo.toml",
+      "fvm_dispatch_tools/Cargo.toml"
+    ]
```

Comment:
Any reason we mention more paths here than below (`sources`)?

Reply:
Good spot, a faulty copy-and-paste. I'll get that fixed.